### PR TITLE
1.21.1rc1

### PR DESCRIPTION
**Pull request by julian-smith-artifex-com**

### CODE REVIEW

**Comment by julian-smith-artifex-com** [comment]
I have read the CLA Document and I hereby sign the CLA

File: changes.txt [hunk 1]
```diff
@@ -1,8 +1,41 @@
...
 ==========
 
+**Changes in Version 1.21.1rc1 (2022-12-12)**
 
-**Changes in Version 1.21.0**
+* This release uses ``MuPDF-1.21.1``.
+
+* Bug fixes:
+
+  * **Fixed** `#2110 <https://github.com/pymupdf/PyMuPDF/issues/2110>`_: Fully embedded font is extracted only partially if it occupies more than one objects
```
Comment (jamie-lemon) [inline changes.txt:10]: objects -> object

**Review by JorjMcKie — APPROVED** [review]

File: docs/changes.rst [hunk 2]
```diff
@@ -3,8 +3,41 @@
...
 ==========
 
+**Changes in Version 1.21.1rc1 (2022-12-12)**
 
-**Changes in Version 1.21.0**
+* This release uses ``MuPDF-1.21.1``.
+
+* Bug fixes:
+
+  * **Fixed** `#2110 <https://github.com/pymupdf/PyMuPDF/issues/2110>`_: Fully embedded font is extracted only partially if it occupies more than one objects
```
Comment (jamie-lemon) [inline docs/changes.rst:12]: object -> objects

**Comment by JorjMcKie** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by JorjMcKie** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**